<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QMainWindow" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>41</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="filter_gb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QCheckBox" name="filter_ckb">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>筛选文件类型</string>
     </property>
    </widget>
    <widget class="QToolButton" name="confirm_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QComboBox" name="filter_cb">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>441</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="file_btn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>只看文件</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dir_btn">
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="text">
         <string>只看文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>751</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="filePath_lab">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>701</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>文件路径：</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>701</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="fileName_lab">
        <property name="text">
         <string>文件名:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filesize_lab">
        <property name="text">
         <string>文件大小:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filetype_lab">
        <property name="text">
         <string>文件类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="parent_lab">
        <property name="text">
         <string>节点:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="parent_ckb">
        <property name="text">
         <string>节点是否是目录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen"/>
  </widget>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/icons/folder1.bmp</normaloff>:/image/icons/folder1.bmp</iconset>
   </property>
   <property name="text">
    <string>设置根目录</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;设置根目录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
